packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import moment from 'moment-timezone';
+import getDisplayName from '../../utils/get-display-name';
+
+type AdditionalProperties = { [key: string]: unknown };
+type TRawUser = {
+  id: string;
+  email: string;
+  firstName: string;
+  lastName: string;
+  language: string;
+  timeZone?: string;
+} & AdditionalProperties;
```

Comment:
This is to ensure that the `props.user` can contain additional properties, but we are only interesting in the ones defined here above

Reply:
oh that's cool